R/readWQPdots.R

```diff
@@ -18,19 +18,22 @@ readWQPdots <- function(...){
   names(values)[names(values) == "siteNumber"] <- "siteid"
   names(values)[names(values) == "siteNumbers"] <- "siteid"
   
+  names(values)[names(values) == "stateCd"] <- "statecode"
   if("statecode" %in% names(values)){
     stCd <- values["statecode"]
-    if(!grepl("US:",stCd)){
-      values["statecode"] <- paste0("US:",stateCdLookup(stCd, "id"))
+    stCdPrefix <- "US:"
+    if(!grepl(stCdPrefix, stCd)){
+      values["statecode"] <- paste0(stCdPrefix, stateCdLookup(stCd, "id"))
     }
   }
   
-  if("stateCd" %in% names(values)){
-    stCd <- values["stateCd"]
-    if(!grepl("US:",stCd)){
-      values["stateCd"] <- paste0("US:",stateCdLookup(stCd, "id"))
+  names(values)[names(values) == "countyCd"] <- "countycode"
+  if("countycode" %in% names(values)){
+    if(!grepl(values["statecode"], values["countycode"])){
+      stCd <- gsub("US:", "", values["statecode"])
+      values["countycode"] <- paste0(values["statecode"], 
+                                     countyCdLookup(stCd, values["countycode"], "id"))
```

Comment:
Oh it was in the previous code with my prefix object, but guess I didn't add it back in appropriately. It didn't seem to make a difference though...